src/mds/LogEvent.h

```diff
@@ -53,11 +53,11 @@ class EMetaBlob;
 // generic log event
 class LogEvent {
 public:
-  friend class MDLog;
   typedef __u32 EventType;
+  friend class MDLog;
 
-  LogEvent() = delete;
   explicit LogEvent(int t) : _type(t) {}
+  LogEvent() = delete;
```

Comment:
Why this change necessary?

Reply:
It is not necessary. I will revert the change.